### PR TITLE
Add systemd service

### PR DESCRIPTION
**Pull request by blavka**

### CODE REVIEW

File: systemd/bc-bridge.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=BigClown bridge
```
Comment (hubpav) [inline systemd/bc-bridge.service]: Use the same name description as reported by --help

File: systemd/bc-bridge.service [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Requisite=dev-bcbridgehid0.device
+
+[Service]
+ExecStart=/usr/bin/bc-bridge --host localhost --prefix nodes/bridge/0
+User=bigclown
+Restart=on-failure
+RestartSec=10s
+
+[Install]
+WantedBy=dev-bcbridgehid0.device
```
Comment (hubpav) [inline systemd/bc-bridge.service]: Missing newline at the end of file, please use .editorconfig plugin for your editor.

File: systemd/prerm [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+systemctl disable bc-bridge.service
+systemctl stop bc-bridge.service
```
Comment (hubpav) [inline systemd/prerm]: Missing newline at the end of file, please use .editorconfig plugin for your editor.

File: systemd/bc-bridge.service (hunk 2 above)
Comment (blavka) [inline systemd/bc-bridge.service]: ok

File: systemd/prerm (hunk 3 above)
Comment (blavka) [inline systemd/prerm]: ok

File: systemd/bc-bridge.service (hunk 1 above)
Comment (blavka) [inline systemd/bc-bridge.service]: ok

**Pull request merged**